### ISSUE TITLE
Podcast 3rd-party links screenreader bug

### ISSUE THREAD

**Issue by RichardPK** [bbc/simorgh#8970]
**Describe the bug**
- Screen reader behaviour on Talkback Android & Voiceover iOS / Mac isn't as expected.

### Talkback Android
- Links are selected multiple times, and read out in the following order (video (sorry to the OSS community) https://bbcnews.slack.com/archives/CJT0284G2/p1615299774170600):

```
1. Spotify [title] link - (link goes to Spotify)
2. Spotify - (link goes to Spotify)
3. [title] - (link goes to Apple)
4. Apple [title] link - (link goes to Apple)
5. Apple - (link goes to Apple)
6. [title] - (link goes to RSS)
7. RSS [title] link -  (link goes to RSS)
8. RSS -  (link goes to RSS)
9. [title] -  (link goes to RSS)
```

### Voiceover iOS 
- H2 read out twice on one swipe (might not be fixeable, could be specific to Voiceover)
- Each link must be swiped twice:
> first swipe - ‘[3rd party title], link’
> second swipe - ‘, [podcast brand], link’
- video (https://bbcnews.slack.com/archives/CJT0284G2/p1615299459167900)
- According to the [Screenreader UX ](https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/6012e39f24aa8f014abe0484)" Announced Text: [Link text], [Brand TItle]" - these should not be separated. 

This appears logical given the html structure, and this is likely connected to the issue on talkback. Might giving the container span the attribute "role=text" solve this?
![Screenshot 2021-03-10 at 11 10 25](https://user-images.githubusercontent.com/9042672/110620706-3dae3480-8191-11eb-83af-e757f645a208.png)

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Additional context**

**Comment by LilyL0u** [comment]
Tested on Android Talkback - each element in the list is only read out once now

**Comment by greenc05** [comment]
@LilyL0u please could you verify with NVDA and FF that the links are as expected please? (See comment in cell 4/F - last tab). Looks ok with Chrome and NVDA (my FF doesn't work).

**Comment by joshcoventry** [comment]
@LilyL0u Was this verified in the end? cc @greenc05

**Comment by LilyL0u** [comment]
> @LilyL0u Was this verified in the end? cc @greenc05

I don't remember, but I can check again
